app/controllers/foreman_virt_who_configure/api/v2/configs_controller.rb

```diff
@@ -52,7 +52,7 @@ def deploy_script
             param :hypervisor_password, String, :desc => N_("Hypervisor password, required for all hypervisor types except for libvirt"), :required => false
             param :satellite_url, String, :desc => N_("Satellite server FQDN"), :required => true
             param :debug, :bool, :desc => N_("Enable debugging output")
-            param :proxy, String, :desc => N_('HTTP proxy that should be used for communication between the server on which virt-who is running and the hypervisors and virtualization managers.'), :required => false
+            param :http_proxy_id, Integer, :required => true, :desc => N_('HTTP proxy ID that should be used for communication between the server on which virt-who is running and the hypervisors and virtualization managers.')
```

Comment:
proxy isn't required :)

Reply:
oops didnt get removed with my change, will push up, my fault

Reply:
I don't think it's worthwhile to break the translation here in order to include "ID" in the description - it's kinda self explanatory from the parameter name

Reply:
@jturel fixed both corrections